### ISSUE TITLE
uncertainty calculation in calc_TLLxTxRatio

### ISSUE THREAD

**Issue by SebastienHuot** [R-Lum/Luminescence#78]
### Expected behaviour
LxTx.Error should always be positive
### Observed behaviour
LxTx.Error sometimes returns a negative value
### Running mini example

```r
library(Luminescence)

```
LxTx.Error <- LxTx*((net_LnLx.Error/net_LnLx) + (net_TnTx.Error/net_TnTx)) (line 235)

At the very least, abs() should be used here.

Better yet, the uncertainty calculation should mirror-image the portion found in calc_OSLLxTxRatio

thanks!

**Comment by RLumSK** [comment]
Thanks and done. Besides, what do you mean with 'mirror-image'? Do you mean that 
`calc_TLLxTxRatio()` should work similar to `calc_OSLLxTxRatio()`?

**Comment by SebastienHuot** [comment]
thanks, Sebastian.

for mirror-image, I had a long day, yesterday. How the Lx/Tx uncertainty is calculated should be the same, whether it is OSL or TL. The approach for OSL rely on Galbraith's 2002 paper, along with classic error propagation concepts. For TL, I am scratching my head, at this moment, when I looked at the code!

so, what I suggest is to apply the same approach, in TL, as you did for OSL. If you think I could be of (useful) help with the coding, let me know.

S

**Comment by RLumSK** [comment]
Salut Sébastien, 

ok, I got you: The approach by Galbraith does not work here. We have different effects (e.g., black body radiation) that superimpose the photon counts. Back the days when it was implemented, Christoph went a couple of times to the statistical service at the university to discuss the problem. The solution as implemented currently, reflects those efforts. I guess there is room for improvement though.
